scripts/nodejs-patches/006-windows-virtual-terminal-input-libuv-4688.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/deps/uv/src/win/tty.c b/deps/uv/src/win/tty.c
+index 7e1f15544b17..37bc2c4ace57 100644
+--- a/deps/uv/src/win/tty.c
++++ b/deps/uv/src/win/tty.c
+@@ -58,6 +58,9 @@
+ #ifndef ENABLE_VIRTUAL_TERMINAL_PROCESSING
+ #define ENABLE_VIRTUAL_TERMINAL_PROCESSING 0x0004
+ #endif
++#ifndef ENABLE_VIRTUAL_TERMINAL_INPUT
++#define ENABLE_VIRTUAL_TERMINAL_INPUT 0x0200
```

Comment:
what is the significance of the 0x0200 value? is it just an available value?

Reply:
You can look up the values here: https://learn.microsoft.com/en-us/windows/console/setconsolemode#parameters

This `#define`, like the one above, is added in case the compilation happens on a Windows machine that doesn't have the most recent system headers and doesn't know about the flag yet, so you can still compile a program that can make use of it when run on more recent Windows machines.

That's not a practical concern for mongosh, but for the libuv PR it's relevant.